company-template.el

```diff
@@ -179,7 +217,8 @@ Leave point at the end of the field."
   (let ((last-pos (point)))
     (while (re-search-forward "\\([^,]+\\),?" end 'move)
       (when (zerop (car (parse-partial-sexp last-pos (point))))
-        (company-template-add-field templ last-pos (match-end 1))
+        (company-template-add-field templ last-pos (match-end 1) nil
+                                    #'company-template--clear-c-like-field)
```

Comment:
Probably. I'm not sure if there are additional cases to be handled here. Maybe not.